pulp_rpm/app/constants.py

```diff
@@ -236,3 +236,28 @@
 )
 
 PACKAGES_DIRECTORY = "Packages"
+
+# Types of advisory packages' sum_types
+# Please keep keys by UPPER keys by value as is in
+# https://github.com/rpm-software-management/createrepo_c/blob/master/src/checksum.h#L43-L54
+# and keep both dicts same for performance reason
+ADVISORY_SUM_TYPES_NUMS = {
+    "UNKNOWN": 0,
+    "MD5": 1,
+    "SHA": 2,
+    "SHA1": 3,
+    "SHA224": 4,
+    "SHA256": 5,
```

Comment:
Maybe you can get this data from createrepo_c: getattr(cr, sum_type_str.upper())